addons/hr_expense/models/hr_expense.py

```diff
@@ -847,8 +847,7 @@ def _prepare_payments_vals(self):
                 for attachment in self.message_main_attachment_id]
         }
 
-    def _prepare_move_lines_vals(self):
-        self.ensure_one()
+    def _get_base_account(self):
         account = self.account_id
         if not account:
```

Comment:
i think we can do better here by doing an early return like returning if account and then we can indent all the code to the left